deploy/examples/How_to_resize_volumes

```diff
@@ -0,0 +1,185 @@
+
+(1) Create a cluster rolebinding for service account:
+
+[root@tlx16cp tmp]# kubectl create clusterrolebinding csi-admin --clusterrole=cluster-admin --serviceaccount=kube-system:csi-datera-controller-sa
+clusterrolebinding.rbac.authorization.k8s.io/csi-admin created
+[root@tlx16cp tmp]# 
+
+
+(2) Edit the kube-apiserver manifest and update the feature-gate as shown below:
+
+[root@tlx16cp tmp]# vi /etc/kubernetes/manifests/kube-apiserver.yaml
+
+    - --feature-gates=VolumeSnapshotDataSource=true,ExpandCSIVolumes=true,ExpandInUsePersistentVolumes=true
+
+[root@tlx16cp tmp]#
+[root@tlx16cp tmp]# ps -ef | grep kube-apiserver
+root     26330 26311  5 04:46 ?        00:06:48 kube-apiserver --advertise-address=172.19.1.80 --allow-privileged=true --authorization-mode=Node,RBAC --client-ca-file=/etc/kubernetes/pki/ca.crt --enable-admission-plugins=NodeRestriction --enable-bootstrap-token-auth=true --etcd-cafile=/etc/kubernetes/pki/etcd/ca.crt --etcd-certfile=/etc/kubernetes/pki/apiserver-etcd-client.crt --etcd-keyfile=/etc/kubernetes/pki/apiserver-etcd-client.key --etcd-servers=https://127.0.0.1:2379 --feature-gates=VolumeSnapshotDataSource=true,ExpandCSIVolumes=true,ExpandInUsePersistentVolumes=true --insecure-port=0 --kubelet-client-certificate=/etc/kubernetes/pki/apiserver-kubelet-client.crt --kubelet-client-key=/etc/kubernetes/pki/apiserver-kubelet-client.key --kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname --proxy-client-cert-file=/etc/kubernetes/pki/front-proxy-client.crt --proxy-client-key-file=/etc/kubernetes/pki/front-proxy-client.key --requestheader-allowed-names=front-proxy-client --requestheader-client-ca-file=/etc/kubernetes/pki/front-proxy-ca.crt --requestheader-extra-headers-prefix=X-Remote-Extra- --requestheader-group-headers=X-Remote-Group --requestheader-username-headers=X-Remote-User --secure-port=6443 --service-account-key-file=/etc/kubernetes/pki/sa.pub --service-cluster-ip-range=10.96.0.0/12 --tls-cert-file=/etc/kubernetes/pki/apiserver.crt --tls-private-key-file=/etc/kubernetes/pki/apiserver.key
+
+[root@tlx16cp tmp]# 
+
+
+(3) Ensure the Datera CSI driver yaml from under /datera-csi/deploy/kubernetes/release/1.0/csi-datera-x.x.x is atleast version 1.0.8 or higher.
```

Comment:
Is this path fixed for CSI driver or we should be more intuitive?

Reply:
I changed to read like this:

(3) Ensure you have v1.0.8 or higher of the Datera CSI driver yaml. If you have downloaded the driver from git, it will be under the datera-csi/deploy/kubernetes/release/1.0/ directory.